workspaces/keycloak/plugins/catalog-backend-module-keycloak/src/lib/config.ts

```diff
@@ -96,6 +96,12 @@ export type KeycloakProviderConfig = {
    * Maximum request concurrency to prevent DoS attacks on the Keycloak server.
    */
   maxConcurrency?: number;
+
+  /**
+   * Whether if the API call will return a brief representation for groups.
```

Comment:
```suggestion
   * Whether the API call will return a brief representation for groups or not. Defaults to true.
```

The wording of this is a bit hard, feel free to come up with your own take if you don't like my suggestion 👍

Reply:
We should add the default seeing as there is one here.